app/components/support_interface/application_navigation_component.rb

```diff
@@ -0,0 +1,51 @@
+module SupportInterface
+  class ApplicationNavigationComponent < ViewComponent::Base
+    include ViewHelper
+
+    def initialize(application_form)
+      @application_form = application_form
+    end
+
+    def links
+      [
+        email_log_link,
+        sentry_link,
+        logit_link,
```

Comment:
Can we have these in alphabetical order?

Reply:
Your wish is my command 🙇

Reply:
:heart: